packages/cinterop/src/commonMain/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt

```diff
@@ -225,23 +233,28 @@ expect object RealmInterop {
 
     fun realm_get_col_key(realm: RealmPointer, classKey: ClassKey, col: String): PropertyKey
 
-    fun realm_get_value(obj: RealmObjectPointer, key: PropertyKey): RealmValue
-    fun realm_set_value(obj: RealmObjectPointer, key: PropertyKey, value: RealmValue, isDefault: Boolean)
+    fun MemTrackingAllocator.realm_get_value(obj: RealmObjectPointer, key: PropertyKey): RealmValue?
+    fun realm_set_value(
+        obj: RealmObjectPointer,
+        key: PropertyKey,
+        value: RealmValue,
+        isDefault: Boolean
+    )
     fun realm_set_embedded(obj: RealmObjectPointer, key: PropertyKey): RealmObjectPointer
     fun realm_object_add_int(obj: RealmObjectPointer, key: PropertyKey, value: Long)
 
     // list
     fun realm_get_list(obj: RealmObjectPointer, key: PropertyKey): RealmListPointer
     fun realm_get_backlinks(obj: RealmObjectPointer, sourceClassKey: ClassKey, sourcePropertyKey: PropertyKey): RealmResultsPointer
     fun realm_list_size(list: RealmListPointer): Long
-    fun realm_list_get(list: RealmListPointer, index: Long): RealmValue
-    fun realm_list_add(list: RealmListPointer, index: Long, value: RealmValue)
+    fun MemTrackingAllocator.realm_list_get(list: RealmListPointer, index: Long): RealmValue?
+    fun RealmValue.realm_list_add(list: RealmListPointer, index: Long)
```

Comment:
Why this sudden change to not pass the `RealmValue` as argument? 🤔 ... I am really puzzled